events/giveaways/giveawayEnded.js

```diff
@@ -1,4 +1,5 @@
 const { MessageEmbed, MessageButton, MessageActionRow } = require("discord.js")
+const Discord = require('discord.js');
```

Comment:
Not sure you need this line since it's already pulling in the necessary library functions in the line above?